distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics/Rank_Method.enso

```diff
@@ -0,0 +1,18 @@
+
+## Specifies how to handle ranking of equal values.
+type Rank_Method
+    ## Use the mean of all ranks for equal values.
+    type Average
+
+    ## Use the lowest of all ranks for equal values.
+    type Minimum
+
+    ## Use the highest of all ranks for equal values.
+    type Maximum
+
+    ## Use same rank value for equal values and next group is the immediate
+       following ranking number.
+    type Dense
+
+    ## Equal values are assigned the next rank in order that they occur.
+    type Ordinal
```

Comment:
This explanation is much better, but I'm still thinking if it would be worth to maybe offer some examples? Not sure if here or next to some method using this. But I'm still not sure if I correctly understand how `Dense` or `Average`  work